pkg/commands/geo_search.ts

```diff
@@ -130,8 +130,8 @@ export class GeoSearchCommand<
         ...(opts?.withHash ? ["WITHHASH"] : []),
       ],
       {
-        ...commandOptions,
         deserialize: transform,
+        ...commandOptions,
```

Comment:
Don't this override deserialize with default value, since we spread right after `deserialize: transform`?

Reply:
`deserialize` is optional in `commandOptions`, so it doesn't overwrite the `deserialize: transform`

user can write their own deserialize method in `commandOptions` to overwrite `transform`. Previously this wasn't possible.

Reply:
I see. I thought `commandOptions` had same default values in it.